Lib/fontbakery/profiles/universal.py

```diff
@@ -174,7 +174,7 @@ def com_google_fonts_check_family_win_ascent_and_descent(ttFont, vmetrics):
         OS/2 and hhea vertical metric values should match. This will produce the
         same linespacing on Mac, GNU+Linux and Windows.
 
-        - Mac OS X uses the hhea values. 
```

Comment:
@miguelsousa what can you say here? ;-)

Reply:
@felipesanches Don't know why Bob thinks the character is unexpected. I'm assuming it's just because he's not aware of this PR #3717

BTW, you never replied to [this question](https://github.com/googlefonts/fontbakery/pull/3717#issuecomment-1101809360), which I posted 4 days ago:
> What do you think about using ⏎ (U+23CE) instead? The character would only exist in the source files; the `unindent_and_unwrap_rationale` method would strip it, thus making it not show up in the reports.

Reply:
@miguelsousa you are correct I had not seen #3717. Seems like a lot of work [yet to be done, it seems] to go through all the Rationale statements and add in the soft return where needed -- or was there only one place such was needed? (From what I can tell there are at present just two occurrences of U+2029 in the repo at present) 

I just noticed the char because VSCode complains:
![image](https://user-images.githubusercontent.com/6842108/164816271-55834c17-efe6-4bf3-ba78-886debf167b5.png) 
when it opens any file containing U+2029.

In any case, that wasn't the most important part of this PR and I'm happy to revert the removal of the soft return. The other diff is more important.

Reply:
@bobh0303 thanks for the info; it makes a strong case for using something other than U+2029.
And you're correct, there are a few other rationale strings that need a "soft return" marker.

Reply:
Submitted #3741 to change the "soft return" marker character.